cmd/admin.go

```diff
@@ -88,57 +84,43 @@ to make automatic initialization process more smoothly`,
 )
 
 func runChangePassword(c *cli.Context) error {
-	if !c.IsSet("password") {
-		return fmt.Errorf("Password is not specified")
-	} else if !c.IsSet("username") {
-		return fmt.Errorf("Username is not specified")
+	if err := argsSet(c, "username", "password"); err != nil {
+		return err
 	}
 
-	setting.NewContext()
-	models.LoadConfigs()
-
-	setting.NewXORMLogService(false)
-	if err := models.SetEngine(); err != nil {
-		return fmt.Errorf("models.SetEngine: %v", err)
+	if err := initDB(); err != nil {
+		return err
 	}
 
 	uname := c.String("username")
 	user, err := models.GetUserByName(uname)
 	if err != nil {
-		return fmt.Errorf("%v", err)
+		return err
 	}
 	user.Passwd = c.String("password")
 	if user.Salt, err = models.GetUserSalt(); err != nil {
-		return fmt.Errorf("%v", err)
+		return err
 	}
 	user.EncodePasswd()
 	if err := models.UpdateUserCols(user, "passwd", "salt"); err != nil {
-		return fmt.Errorf("%v", err)
+		return err
 	}
 
-	fmt.Printf("User '%s' password has been successfully updated!\n", uname)
+	fmt.Printf("%s's password has been successfully updated!\n", user.Name)
 	return nil
 }
 
 func runCreateUser(c *cli.Context) error {
-	if !c.IsSet("name") {
-		return fmt.Errorf("Username is not specified")
-	} else if !c.IsSet("password") {
-		return fmt.Errorf("Password is not specified")
-	} else if !c.IsSet("email") {
-		return fmt.Errorf("Email is not specified")
+	if err := argsSet(c, "username", "password"); err != nil {
```

Comment:
This is breaking change and also (name -> username) and also is missing email check

Reply:
Wooops, looks like I forgot to edit after copypasting - nice catch! (good thing there are code reviews 😌)